packages/renderer/assets/mermaid/mermaid.min.js

```diff
@@ -1351,7 +1351,7 @@ Expecting `+Ft.join(", ")+", got '"+(this.terminals_[Ot]||Ot)+"'":gn="Parse erro
return new Hs},F(Qo,"LayoutMetaDataService/lambda$14$Type",883),D(884,1,My,jZ),l.vg=function(e){return b$(u(e,53))},F(Qo,"LayoutMetaDataService/lambda$15$Type",884),D(885,1,P4,$Z),l.ug=function(){return new O0},F(Qo,"LayoutMetaDataService/lambda$16$Type",885),D(886,1,My,Ac),l.vg=function(e){return P$(u(e,53))},F(Qo,"LayoutMetaDataService/lambda$17$Type",886),D(887,1,P4,zZ),l.ug=function(){return new Bpt},F(Qo,"LayoutMetaDataService/lambda$18$Type",887),D(888,1,My,QT),l.vg=function(e){return VYt(u(e,208))},F(Qo,"LayoutMetaDataService/lambda$19$Type",888),D(871,1,P4,tc),l.ug=function(){return new Pu},F(Qo,"LayoutMetaDataService/lambda$2$Type",871),D(872,1,My,Kc),l.vg=function(e){return new DR(u(e,74))},F(Qo,"LayoutMetaDataService/lambda$3$Type",872),D(873,1,P4,zi),l.ug=function(){return new OC},F(Qo,"LayoutMetaDataService/lambda$4$Type",873),D(874,1,My,zL),l.vg=function(e){return new Lnt(u(e,142))},F(Qo,"LayoutMetaDataService/lambda$5$Type",874),D(875,1,P4,HZ),l.ug=function(){return new Cx},F(Qo,"LayoutMetaDataService/lambda$6$Type",875),D(876,1,My,VZ),l.vg=function(e){return new kwt(u(e,116))},F(Qo,"LayoutMetaDataService/lambda$7$Type",876),D(877,1,P4,sx),l.ug=function(){return new Yu},F(Qo,"LayoutMetaDataService/lambda$8$Type",877),D(878,1,My,km),l.vg=function(e){return new ree(u(e,373))},F(Qo,"LayoutMetaDataService/lambda$9$Type",878);var cft=Ki(qI,"IProperty");D(23,1,{35:1,686:1,23:1,146:1},Xe),l.wd=function(e){return PQe(this,u(e,146))},l.Fb=function(e){return _t(e,23)?hn(this.f,u(e,23).f):_t(e,146)&&hn(this.f,u(e,146).tg())},l.wg=function(){var e;if(_t(this.b,4)){if(e=O3t(this.b),e==null)throw it(new No(ode+this.f+"'. Make sure it's type is registered with the "+(P0(gP),gP.k)+d8t));return e}else return this.b},l.sg=function(){return this.d},l.tg=function(){return this.f},l.ne=function(){return this.i},l.Hb=function(){return Vg(this.f)},l.Ib=function(){return"Layout Option: "+this.f},F(Qo,"LayoutOptionData",23),D(24,1,{},Ue),F(Qo,"LayoutOptionData/Builder",24),D(175,22,{3:1,35:1,22:1,175:1},OM);var Pd,rp,ea,qn,V2,ip=jr(Qo,"LayoutOptionData/Target",175,Hr,van,Ltn),X3e;D(277,22,{3:1,35:1,22:1,277:1},Ox);var La,Oo,us,U6,gc,V1,I7,_St,Q3e=jr(Qo,"LayoutOptionData/Type",277,Hr,Kon,Mtn),Z3e,WS,SSt;D(110,1,{110:1},Fx,ah,v$),l.Fb=function(e){var n;return e==null||!_t(e,110)?!1:(n=u(e,110),Fc(this.c,n.c)&&Fc(this.d,n.d)&&Fc(this.b,n.b)&&Fc(this.a,n.a))},l.Hb=function(){return Az(lt(ct(Qn,1),De,1,5,[this.c,this.d,this.b,this.a]))},l.Ib=function(){return"Rect[x="+this.c+",y="+this.d+",w="+this.b+",h="+this.a+"]"},l.a=0,l.b=0,l.c=0,l.d=0,F(WI,"ElkRectangle",110),D(8,1,{3:1,4:1,8:1,414:1},Ca,Srt,$e,xo),l.Fb=function(e){return fne(this,e)},l.Hb=function(){return p8(this.a)+F1n(p8(this.b))},l.Jf=function(e){var n,r,s,o;for(s=0;s<e.length&&Ure((Rr(s,e.length),e.charCodeAt(s)),f1e);)++s;for(n=e.length;n>0&&Ure((Rr(n-1,e.length),e.charCodeAt(n-1)),d1e);)--n;if(s>=n)throw it(new Fn("The given string does not contain any numbers."));if(o=vy(e.substr(s,n-s),`,|;|\r|
```

Comment:
Please avoid modifying `mermaid.min.js` in this PR — it's auto-generated from the [mermaid.js](https://mermaid.js.org/) library.